lib/check_data_points.rb

```diff
@@ -0,0 +1,9 @@
+class CheckDataPoints
+  def self.tipline_requests(team_id, date = nil)
+  	# date in format `2023-08-23`
+  	date = begin DateTime.parse(date) rescue nil end
```

Comment:
Suggestion: Implement a private method in this class which is a helper to parse/validate input datetime parameters.